community/en/census/tf20_census.ipynb

```diff
@@ -0,0 +1,1422 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "TF 2.0 Census Dataset: Keras, tf.keras.layers + tf.data + save_keras_model",
+      "version": "0.3.2",
+      "provenance": [],
+      "collapsed_sections": [
+        "dsDV5jNzgbGl"
+      ],
+      "toc_visible": true
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    }
+  },
+  "cells": [
+    {
+      "metadata": {
+        "id": "dsDV5jNzgbGl",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "##### Copyright 2019 The TensorFlow Authors."
+      ]
+    },
+    {
+      "metadata": {
+        "id": "Y4UiAxk2gPnp",
+        "colab_type": "code",
+        "cellView": "both",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "#@title Licensed under the Apache License, Version 2.0 (the \"License\");\n",
+        "# you may not use this file except in compliance with the License.\n",
+        "# You may obtain a copy of the License at\n",
+        "#\n",
+        "# https://www.apache.org/licenses/LICENSE-2.0\n",
+        "#\n",
+        "# Unless required by applicable law or agreed to in writing, software\n",
+        "# distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+        "# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+        "# See the License for the specific language governing permissions and\n",
+        "# limitations under the License."
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "mHF9VCProKJN",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "# Census using tf.Keras + TF.data.Dataset in TF 2.0\n",
```

Comment:
Please remove the "in TF 2.0" from this doc. This will just be standard way to do things, soon enough